### DISCUSSION TITLE
new installation and "Error occurred while index vault to vector store"

### DISCUSSION THREAD

**Discussion by rajah9** [logancyang/obsidian-copilot#564] [Q&A]
**Describe how to reproduce**

- Installed obsidian-copilot plugin on Obsidian 1.6.7.
- Set LLM to Claude.
- Entered Anthropic API key.
- Saved and reloaded. 
- verified that the LLM pointed to Claude and that the Anthropic API Key was filled in. 

**Expected behavior**
From the command palette, Chose "Copilot: Index (refresh) vault for QA" . (Also tried "Copilot: Force re-index vault for QA")
Expected the vault to be indexed. 

**Actual behavior**
Got error message: "An error occurred while index vault to vector store."
Happens in Debug mode also. 

**Additional context**

**Comment by logancyang** [comment]
What embedding model was set in your setting? You need to set that too.
